### PR TITLE
Add new supporter to readme docs

### PR DESCRIPTION
**Pull request by pavlenex**
Picture will be visible once [#1174 ](https://github.com/btcpayserver/btcpayserver/pull/1174)merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,12 @@ The BTCPay Server Project is proudly supported by these entities through the [BT
           <span>ACINQ</span>
         </a>
       </td>
+       <td align="center" valign="middle">
+        <a href="https://lunanode.com" target="_blank">
+          <img  src="Bhttps://raw.githubusercontent.com/btcpayserver/btcpayserver/master/BTCPayServer/wwwroot/img/lunanode.svg?sanitize=true" alt="LunaNode" height=100>
+          <br/>
+          <span>Luna Node</span>
```
Comment (NicolasDorier) [inline README.md]: LunaNode is one word

Comment (pavlenex) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -182,6 +182,12 @@ The BTCPay Server Project is proudly supported by these entities through the [BT
           <span>ACINQ</span>
         </a>
       </td>
+       <td align="center" valign="middle">
+        <a href="https://lunanode.com" target="_blank">
+          <img  src="Bhttps://raw.githubusercontent.com/btcpayserver/btcpayserver/master/BTCPayServer/wwwroot/img/lunanode.svg?sanitize=true" alt="LunaNode" height=100>
```
Comment (NicolasDorier) [inline README.md]: source not good, why there is a B ?

File: README.md [hunk 3]
```diff
@@ -182,6 +182,12 @@ The BTCPay Server Project is proudly supported by these entities through the [BT
           <span>ACINQ</span>
         </a>
       </td>
+       <td align="center" valign="middle">
+        <a href="https://lunanode.com" target="_blank">
```
Comment (NicolasDorier) [inline README.md]: lowercase

File: README.md (hunk 2 above)
Comment (pavlenex) [reply README.md]: ugh fuck

File: README.md (hunk 3 above)
Comment (pavlenex) [reply README.md]: it should be lowercase it's a goddamn link

**Comment by pavlenex** [comment]
Depends on [#1176](https://github.com/btcpayserver/btcpayserver/pull/1176)

**Pull request merged**